book.rb

```diff
@@ -0,0 +1,18 @@
+class Book
+  attr :title, :author
```

Comment:
- [ ] You are expected to have both **_Setters_** and **_getters_** for instance variables (remember about attr_accessor).
This `attr :title, :author` is similar to `attr_reader`, which is only **_getter_**. 

```ruby
attr_accessor :title, :author, :rentals
```